docs/wiki/ModulesUsage.md

```diff
@@ -75,3 +85,115 @@ $inputObject = @(
 )
 az deployment group create @inputObject
 ```
+
+## Orchestrate deployment
+
+This section shows you how you can orchestrate a deployment using multiple resource modules
+
+### Template-orchestration
+
+The _template-orchestrated_ approach means using a _main_ or so-called _master template_ for deploying resources in Azure. The _master template_ will only contain nested deployments, where the modules – instead of embedding their content into the _master template_ – will be linked from the _master template_.
+
+With this approach, modules need to be stored in an available location, where Azure Resource Manager (ARM) can access them. This can be achieved by storing the modules templates in an accessible location location like _template specs_ or the _bicep registry_.
+
+In an enterprise environment, the recommended approach is to store these templates in a private environment, only accessible by enterprise resources. Thus, only trusted authorities can have access to these files.
```

Comment:
Do we imply here that customers should fork our repo and deploy it to their own env?

Reply:
It's one of the 3 scenarios with decribe in the `Getting started` section. 
It's supposed to imply that an enterprise should, if they want to use the repo (including the pipelines) not publish their templates to a public space like the public bicep registry (once available)